driver/remote/driver.go

```diff
@@ -88,11 +88,19 @@ func (d *Driver) Client(ctx context.Context) (*client.Client, error) {
 }
 
 func (d *Driver) Features() map[driver.Feature]bool {
+	var historyAPI bool
+	ctx := context.Background()
```

Comment:
Hm, since `Features` now will pretty much *always* make external requests, it feels like we should probably just modify the interface to take a `Context` instead of using the background? It's not great that this operation wouldn't cancel properly, we also do it from inside `toSolveOpt`, so we've always got a context there.

Reply:
Ignore me. That's a terrible idea - if the call fails, the feature should just be marked as false.